packages/core/src/domain/document-query/trigger-and-query.ts

```diff
@@ -65,12 +68,13 @@ export abstract class TriggerAndQueryDocRefs {
     } = config;
     let docsFound = 0;
 
+    // eslint-disable-next-line @metriport/eslint-rules/no-named-arrow-functions
```

Comment:
If we add this we'll never be notified to modify this again. The change is pretty simple and doesn't change the functionality of the code, so maybe we can just update it?

```ts
async function triggerDQAndCheckResults(...) {
   ...
}
```

Reply:
Its not that straight forward because of the this references within the arrow function